ui1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="author" content="Ozan Paker">
+    <meta name="keywords" content="html pratikleri,html">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML-Practices</title>
+    <!-- title ne işe yarar? -->
+    <link id="favicon" rel="shortcut icon" type="image/png" href="favicon.png"/>
+  </head>
+  <body>
+    <h1>Html Öğrenme</h1>
```

Comment:
Bu başlıklara acaba bi başlık mı atsak ne için kullanıldığı da belli olur 😅 başlık tipleri  ve gösterimleri gibi bir şey olabilir